waku/v2/node/wakunode2.nim

```diff
@@ -393,9 +393,9 @@ proc mountFilter*(node: WakuNode) =
 
 # NOTE: If using the swap protocol, it must be mounted before store. This is
 # because store is using a reference to the swap protocol.
-proc mountSwap*(node: WakuNode) =
-  info "mounting swap"
-  node.wakuSwap = WakuSwap.init(node.peerManager, node.rng)
+proc mountSwap*(node: WakuNode, swapConfig: SwapConfig = SwapConfig()) =
```

Comment:
Now that you have an `init` for `SwapConfig` you could do something like:

```suggestion
proc mountSwap*(node: WakuNode, swapConfig = SwapConfig.init()) =
```

Reply:
Thanks. Made the changes. 

But what is the difference between having `SwapConfig()` this way and specifying `init` in `SwapConfig.init()`?
Both seem to work.

Reply:
An empty constructor, like `SwapConfig()` will initialise each field with the default value for that _type_ (e.g. `paymentThreshold`, `disconnectThreshold` will be `0`, as that is the default for `int`). `init()` will construct a `SwapConfig` object by calling the `init()` function you defined, in other words use the default values as specified by you.

Reply:
Thanks. This was helpful.